interpreter/cling/lib/Interpreter/LookupHelper.cpp

```diff
@@ -1729,16 +1729,22 @@ namespace cling {
                                                       llvm::StringRef funcName,
                                                       llvm::StringRef funcProto,
                                                       DiagSetting diagOnOff,
-                                                      bool objectIsConst) const{
+                                                      bool objectIsConst) {
     assert(scopeDecl && "Decl cannot be null");
 
-    return execFindFunction<ParseProto>(*m_Parser, m_Interpreter,
+    auto it = findFunctionProtoCache.find({std::string(funcName), std::string(funcProto)});
```

Comment:
Can we create the pair outside of the call?  This would avoid having to recreate/copy the std::string a 2nd time at the end of the routine.